tech-test/src/tests/Search.test.js

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import Search from "../components/Search";
+
+describe("Search", () => {
+
+  const validProps = {
+    setSearchResults: () => {}
+  };
+
+  it("renders Search component correctly", () => {
+    const { asFragment } = render(
+    <Search 
+      setSearchResults={validProps.setSearchResults} 
+    />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders button element", () => {
```

Comment:
Could a test be added to make sure the function is called when the button is clicked? The `fireEvent` method would be useful here